components/brave_wallet/browser/tx_service.cc

```diff
@@ -440,4 +452,8 @@ void TxService::ProcessFilHardwareSignature(
       chain_id, tx_meta_id, signed_message, std::move(callback));
 }
 
+TxStorageDelegate* TxService::GetDelegateForTesting() {
+  return static_cast<TxStorageDelegate*>(delegate_.get());
```

Comment:
Will that work without static_cast?

Reply:
yes I will remove it.

Reply:
fixed in 330c37ab4a9a518d77a58769424ffe8b45e84776